pkg/defaults/defaults.go

```diff
@@ -53,7 +53,7 @@ const (
 	DefaultRegistryPort         = 5050
 
 	//tags, versions, repos
-	DefaultEVETag               = "10.11.0" // DefaultEVETag tag for EVE image
+	DefaultEVETag               = "11.7.0" // DefaultEVETag tag for EVE image
```

Comment:
FYI @milan-zededa seems moving to the new tag broke EdenGCP tests (3 attempts full [failed](https://github.com/lf-edge/eden/actions/runs/7568041549) vs previous full [green](https://github.com/lf-edge/eden/actions/runs/7474836301)).

Reply:
Can't be that we ran out of space, can it? https://github.com/lf-edge/eden/actions/runs/7568041549/job/20647150138#step:17:1641

Reply:
I will have a look at this early next week.